packages/model-viewer/src/features/annotation.ts

```diff
@@ -124,6 +132,46 @@ export const AnnotationMixin = <T extends Constructor<ModelViewerElementBase>>(
       this[$needsRender]();
     }
 
+    /**
+     * This method returns in-scene data about a requested hotspot including
+     * its position in screen (canvas) space and its current visibility.
+     */
+    queryHotspot(name: string): HotspotData | null {
+      const hotspot = this[$hotspotMap].get(name);
+      if (hotspot == null) {
+        return null;
+      }
+
+      const position = toVector3D(hotspot.position);
+      const normal = toVector3D(hotspot.normal);
+      const facingCamera = hotspot.facingCamera;
+
+      const scene = this[$scene];
+      const camera = scene.getCamera();
+      const vector = new Vector3();
+
+      vector.setFromMatrixPosition(hotspot.matrixWorld);
+      vector.project(camera);
+
+      const widthHalf = scene.width / 2;
+      const heightHalf = scene.height / 2;
+
+      vector.x = (vector.x * widthHalf) + widthHalf;
+      vector.y = -(vector.y * heightHalf) + heightHalf;
+
+      const screenPosition = toVector2D(
```

Comment:
Regarding U,V - would it be better to return a `Vector3D` instead? It would be an OpenGL-style screen-space depth which most users would probably ignore, but it might still be useful for something.

Reply:
Yep, that seems reasonable! The depth is a bit of a weird number presented raw because I'm pretty sure it's some sort of logarithmic scale (so it's almost always ~0.98) but it could still be used to sort hotspots to affect overlay draw order or something